tests/qiskit_qir/test_qiskit_to_qir.py

```
@@ -0,0 +1,363 @@
+
+class TestQiskitToQir:
+    """Tests for the qiskit_to_qir function with sequential verification."""
+
+    def test_bell_circuit(self, bell_circuit):
+        module = qiskit_to_qir(bell_circuit)
+        func = _get_entry_point(module)
+        assert required_num_qubits(func) == 2
+        assert required_num_results(func) == 2
+
+        body = _get_body(module)
+        gate_ops = [l for l in body if "qis__" in l]
```

Ruff (0.15.13)

[error] Ambiguous variable name: `l`

(E741)